src/renderer/shared/api/network/service/chainsService.ts

```diff
@@ -162,3 +162,24 @@ function sortChainsByBalance(
     sortBy(testnets.noBalance, 'name'),
   );
 }
+
+const sumValues = (firstValue?: string, secondValue?: string): string => {
```

Comment:
I find it pretty weird that both sum functions moved down in `network/chainsService`
Will they ever be used anywhere else besides Assets page?
If not then `pages/Assets/*-utils` is still a good place or the new one `AssetsChainView` 😉

Reply:
I moved it there because we use `sumBalance` in chainsService and `sumValues` is a helper for `sumBalance`